dashboard/app/helpers/race_interstitial_helper.rb

```diff
@@ -5,7 +5,7 @@ def self.show?(user)
     return false if user.races
     return false if user.teacher?
     return false if user.under_13?
-    return false if user.account_age_days < 7
+    return false if user.days_since_first_sign_in && user.days_since_first_sign_in < 7
```

Comment:
Is it worth adding a test to `dashboard/test/helpers/race_interstitial_helper_test.rb` to test the case we're fixing here?

Reply:
Did not realize we had tests there, which are now failing 😄 . Will update!

Reply:
Updated, give it a look when you have a second!